File: Mozu.SiteBuilder.UX.Admin/Api/Models/Account/AccountInformation.cs


namespace Mozu.SiteBuilder.UX.Admin.Api.Models.Account
{
    
    public class AccountInformation
    {

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Email { get; set; }

        public string OldPassword { get; set; }

        public string NewPassword { get; set; }

        public string ConfirmPassword { get; set; }
    }
}

File: Mozu.SiteBuilder.UX.Admin/Scripts/app/model/AccountInformation.js
/**
 * @class Taco.model.AccountInformation
 */

	Ext.define('Taco.model.AccountInformation', {
	    extend: 'Taco.core.data.Model',
	    fields: [
	        { name: 'email',           type: 'string' },
	        { name: 'firstName',       type: 'string' },
	        { name: 'lastName',        type: 'string' },
	        { name: 'newPassword',     type: 'string' },
	        { name: 'oldPassword',     type: 'string' },
	        { name: 'confirmPassword', type: 'string' }
	    ]
	});
